### ISSUE TITLE
[Bug] 1.11.2 Manual doesn't open

### ISSUE THREAD

**Issue by Dragnier** [rwtema/extrautilities_old_issues#1278]
I made the Extra Utilities 2 Manual and it doesn't open on right-click at all.

**Comment by rob07allen** [comment]
Not sure if bumping helps on here, but *BUMP*

Unless someone knows where the recipes for Extra Utilities 2 is online, this mod is almost unusable without the manual.

**Comment by InjuredWolf** [comment]
you could always use JEI. There are recipes there, and most items have documentation in JEI, too.

**Comment by rob07allen** [comment]
Wow didn't know about that mod. That's super useful! Thanks!

**Comment by InjuredWolf** [comment]
JEI basically replaced NEI because Chickenbones didn't update for ages. Most mods have compatibility with JEI so it's advised you use that. ExUtils2 has descriptions for its items within JEI.

**Comment by Dragnier** [comment]
Yes, recipe wise that would work. However, there are occasionally informational things about the mod that are in the manual besides recipes that it would be nice to have access to without needing to alt-tab or search. Especially when it may be something that isn't up-to-date on wikis.

**Comment by NatalieBee** [comment]
Hi, also having this problem. Thanks.

**Comment by ShelLuser** [comment]
Just wanted to mention that this is still an ongoing issue. In the mean time (between opening of this issue and now) there have been a few updates with Forge (1.11.2 has become the default, and the current version being 2259) and unfortunately nothing changed. I took the liberty of pointing out a duplicate issue.

**Comment by mgwerner** [comment]
Still an issue, Forge 13.20.0.2300, extrautils2-1.11.2-1.4.4

**Comment by InjuredWolf** [comment]
Does the manual have any info that the JEI descriptions don't?

**Comment by tuxntosh** [comment]
Still an issue in Single Player and on Server, Forge 13.20.1.2476, extrautils2-1.11.2-1.6.3

Other Mods installed:

airbreathercore-1.0.2-mc1.11.jar
antiqueatlas-1.11.2-4.4.6.jar
AppleCore-mc1.11.2-2.2.2.jar
AppleSkin-mc1.11-1.0.8.jar
AutoSapling 1.11.2-1.3.5.jar
backpacks 1.11.2 - 3.3.2.jar
BetterFoliage-MC1.11.2-2.1.10.jar
BiblioCraft[v2.3.3][MC1.11.2].jar
BiblioWoods[BiomesOPlenty][v2.0][MC1.11.2].jar
BiomesOPlenty-1.11.2-6.2.0.2254-universal.jar
Chameleon-1.11.2-3.1.0.jar
Chisel-MC1.11.2-0.0.14.25.jar
CodeChickenLib-1.11.2-3.0.0.298-universal.jar
CTM-MC1.11.2-0.2.2.12.jar
CustomNPCs_1.11.2(21jul17).jar
Decocraft-2.4.2_1.11.2.jar
extrautils2-1.11.2-1.6.3.jar
GraveStone-1.11.2-Graves-1.0.10.1.jar
InventoryTweaks-1.62.jar
jei_1.11.2-4.5.0.294.jar
Mantle-1.11.2-1.2.0.26.jar
Mob-Totems-Mod-1.11.2.jar
mystcraft-1.11.2-0.13.1.00.jar
NotEnoughItems-1.11.2-2.3.0.227-universal.jar
OpenBlocks-1.11.2-1.7.2.jar
OpenModsLib-1.11.2-0.11.2.jar
Pam's HarvestCraft 1.11.2h.jar
redstonepaste-mc1.11.2-1.7.4.jar
Ruins-1.11.2.jar
signpost-1.11.2-1.05.5.jar
SoundFilters-0.10_for_1.11.jar
StorageDrawers-1.11.2-4.2.10.jar
StorageDrawersExtras-1.11.2-2.0.0.jar
TConstruct-1.11.2-2.7.1.34.jar
TinkerToolLeveling-1.11.2-1.0.1.jar
ToolBelt-1.11.2-1.7.1.jar
UndergroundBiomesConstructs-1.11-1.0.v.19.jar
VerticalSlabs-1.1.jar
Waila-1.7.1_1.11.2.jar
yalsm-2.1.7-mc1.11.jar

**Comment by Immaculala** [comment]
15th November 2017. Single Player. Forge 13.20.1.2386  extrautils2-1.11.2-1.6.5  This issue still exists :(

Yes you can look up recipes in JEI but a manual does more than just give recipes - I need the manual as a guide to the mod and where to start.

**Comment by stresser1** [comment]
ditto, please fix the book!

**Comment by KAWGAMING** [comment]
Does anyone know where we can find a manual online?

**Comment by Immaculala** [comment]
That would help me too

**Comment by tuxntosh** [comment]
Post from Thread https://github.com/rwtema/extrautilities/issues/1361:

Dgppol commented 8 months ago • edited 8 months avi
@Decrepitdoors
I used an older version of MC to create an album of the manual: http://imgur.com/a/bu4rL

**Comment by Immaculala** [comment]
Thanks tuxntosh for tracking that down.
Huge thanks to Decrepitdors for doing the work.

But I'm still totally hacked off it's not fixed in game.
